src/StimulusBundle/doc/index.rst

```diff
@@ -39,6 +39,10 @@ necessary files. If not, or you're curious, see :ref:`Manual Setup <manual-insta
     If you're using Encore, be sure to install your assets (e.g. ``npm install``)
     and restart Encore.
 
+.. note::
+
+    Alternatively, the `@symfony/stimulus-bundle package`_ can be used to install the JavaScript assets without requiring PHP.
```

Comment:
Are we sure everything works here ? :| This feels suspicious as some path are hard-coded for controller.json

Reply:
I will double check later

Reply:
Well, you're right, those `dist/` files are only needed when you use the AssetMapper, since we tell users to use npm package `@symfony/stimulus-bridge` when using Encore.

Reply:
I will mark this npm package as private, and remove it from npm. I will adapt the `README.md` telling people looks for assets must install `@symfony/stimulus-bridge` if using Encore. 

Let's see in the future if we really need to publish this one.

Thanks!